addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -143,6 +143,11 @@ export class AttachmentViewer extends Component {
         const attachmentViewer = this.attachmentViewer;
         const refs = this._getRefs();
         const image = refs[`image_${this.attachmentViewer.attachment.id}`];
+        // some actions are too fast that sometimes this function is called
+        // before setting the refs, so we just do nothing when image is null
+        if(!image){
```

Comment:
space before "(" and after ")" (I think it can be done automatically with https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines)